src/main/java/org/folio/rest/impl/PutOrdersByIdHelper.java

```diff
@@ -99,10 +97,21 @@ public void updateOrder(String orderId, CompositePurchaseOrder compPO) {
   public CompletableFuture<Void> openOrder(CompositePurchaseOrder compPO) {
     compPO.setWorkflowStatus(OPEN);
     compPO.setDateOrdered(new Date());
-    return updateInventory(compPO)
-      .thenCompose(this::updateOrderSummary);
+    return fetchCompositePoLines(compPO)
+      .thenCompose(v -> changePoLineReceiptStatuses(compPO))
```

Comment:
Instead of `thenCompose` you might use `thenAccept`. In this case `changePoLineReceiptStatuses` will not return `CompletionStage` but just `void`.

Reply:
fixed